src/frontend/src/scheduler/execution/query.rs

```diff
@@ -226,7 +226,9 @@ impl QueryRunner {
                     }
                 }
                 _ => {
-                    unimplemented!()
+                    return Err(ErrorCode::NotImplemented(
+                    "".to_string(), None.into(),
```

Comment:
We may a provide some useful messages here.

Reply:
add message for the all change.